bin/start-smart.sh

```diff
@@ -64,7 +64,7 @@ ZEPPELIN_LOGFILE="${SMART_LOG_DIR}/zeppelin-${SMART_IDENT_STRING}-${HOSTNAME}.lo
 LOG="${SMART_LOG_DIR}/smart-cli-${SMART_IDENT_STRING}-${HOSTNAME}.out"
 
 SMART_SERVER=org.smartdata.server.SmartDaemon
-JAVA_OPTS+=" -Dzeppelin.log.file=${ZEPPELIN_LOGFILE}"
+JAVA_OPTS+=" -Dzeppelin.log.file=${ZEPPELIN_LOGFILE} -Dzeppelin.conf.dir=${SMART_CONF_DIR}"
```

Comment:
1. you may forgot to set value for smart.conf.dir through -D
2. better not to hard code zeppelin.conf.dir=${SMART_CONF_DIR}, we can define a variable like SMART_ZEPPELINE_CONF_DIR, if it is not specified then we use ${SMART_CONF_DIR} by default